packages/embeds/embed-core/src/Inline/inlineHtml.ts

```diff
@@ -1,5 +1,5 @@
-const html = `<div id="wrapper" style="top:50%; left:50%" class="absolute z-highest">
-<div style="transform:translate(-50%,-50%)"  class="loader border-brand dark:border-darkmodebrand">
```

Comment:
style="top:50%; left:50%" and style="transform:translate(-50%,-50%)" has to be on same element for center positioning to work correctly.